### ISSUE TITLE
What pretraining tasks were used for training the teacher model?

### ISSUE THREAD

**Issue by Dinghaoxuan** [Microtreei/TSD-SR#13]
Hello,

Thank you for providing the pretrained weights of the teacher model. Could you please clarify:
What pretraining tasks were used for training the teacher model?

This information would help us better understand the model's capabilities and potential applications.

**Comment by Microtreei** [comment]
Thanks for your attention.
We just use diffusion loss to train the teacher on HR datasets. Our original intention is to reduce the generation of some unrealistic styles (e.g. cartoon, line drawing...) in the T2I task. This is done to make the teacher model more adaptable to the real-world image restoration task (Because we hope that the teacher model has capability to learn the denoising trajectory of GT here, not unrealistic styles).

**Comment by AwakenLee** [comment]
> Thanks for your attention. We just use diffusion loss to train the teacher on HR datasets. Our original intention is to reduce the generation of some unrealistic styles (e.g. cartoon, line drawing...) in the T2I task. This is done to make the teacher model more adaptable to the real-world image restoration task (Because we hope that the teacher model has capability to learn the denoising trajectory of GT here, not unrealistic styles).

Any plans to release the training code for the teacher model? I’m having trouble reproducing the results. Thanks!
